Cargo.toml

```diff
@@ -7,11 +7,12 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-anyhow  = { version = "1.0.31" }
-chrono  = { version = "0.4.13" }
-futures = { version = "0.3.5"  }
-tokio   = { version = "0.2.22", features = ["rt-core", "sync", "macros", "time"] }
+anyhow      = { version = "1.0.31" }
+chrono      = { version = "0.4.13" }
+futures     = { version = "0.3.5"  }
+lazy_static = { version = "1.4.0" }
```

Comment:
Personally I've become a fan of using [`once_cell`](https://lib.rs/crates/once_cell) instead of `lazy_static` but they do pretty much the same thing so it's up to you.

I like `once_cell` because it's possible to put the static inside the method where it is used and it's a little clearer what is happening because there isn't the macro syntax.

Reply:
Oh, I prefer not to use macros when possible, so sure... I'll do the change

Reply:
I imagine you have your fair bit of experience with macros from Clojure, I personally have abused them before, and, I tend to avoid them... necessary in some instances, but not something to use for _everything_

Reply:
Actually, I'll do this on a different PR, this one is getting scope creep, apologies

Reply:
Yeah, I think I'm more comfortable with macros than most perhaps, but I agree, when it's not necessary, using a different tool is better.